docs/02-app/01-building-your-application/06-optimizing/11-static-assets.mdx

```diff
@@ -8,22 +8,26 @@ description: Next.js allows you to serve static files, like images, in the publi
 
 Next.js can serve static files, like images, under a folder called `public` in the root directory. Files inside `public` can then be referenced by your code starting from the base URL (`/`).
 
-For example, if you add `me.png` inside `public`, the following code will access the image:
+For example, the file `public/avatars/me.png` can be viewed by visiting the `/avatars/me.png` path. The code might look like:
 
-```jsx filename="Avatar.js"
+```jsx filename="Example.js"
 import Image from 'next/image'
 
-export function Avatar() {
-  return <Image src="/me.png" alt="me" width="64" height="64" />
+export function Avatar({ id, alt }) {
+  return <Image src={`/avatars/${id}.png`} alt={alt} width="64" height="64" />
+}
+
+export function AvatarOfMe() {
```

Comment:
Why have two separate components here?

Reply:
I'm demonstrating a dynamic `src` string here.

The previous example wasn't clear why you would use `<Image src="/me.jpg" />` vs `import me from './me.jpg; <Image src={me} />` so the new example makes it clear that it works with dynamic url string.